console-subscriber/src/builder.rs

```diff
@@ -456,6 +461,60 @@ impl Builder {
     }
 }
 
+/// Specifies the address on which a [`Server`] should listen.
+///
```

Comment:
This type's documentation should probably include something like:
```suggestion
///
/// This type is passed as an argument to the [`Builder::server_addr`]
/// method, and may be either a TCP socket address, or a [Unix Domain Socket]
/// (UDS) address. Unix Domain Sockets are only supported on Unix-compatible
/// operating systems, such as Linux, BSDs, and macOS.
///
/// [Unix Domain Socket]: https://en.wikipedia.org/wiki/Unix_domain_socket 
```

Reply:
Done.